lib/checks.js

```diff
@@ -23,37 +25,37 @@ function failStateCheck(callee, message, formatArgs) {
     failCheck(errors.IllegalStateError, callee, message, formatArgs);
 }
 
-module.exports.checkArgument = function(value, message) {
+module.exports.checkArgument = function checkArgument(value, message) {
```

Comment:
Yeah, might as well do that at this point. Will also separate the internal name from the exported one.